In [22]:
from sympy import *

# Chapter 4 - Problems
## Problem 1.
![image.png](../../utils/kinematics-assignments-images/problem-1-2.png)

### Answer
The first point is where the ball moves the quickes, then it is slowest in point two and in the third point it is quicker again. So the answer is E.

## Problem 2
![image.png](../../utils/kinematics-assignments-images/problem-2-2.png)

### Answer
We know that the ball undergoes protejctile motion after being kicked horizontally. 
The given staircase consists of square steps with side length $d$, meaning the ball falls a vertical distance $d$ while traveling a horizontal distance $d$.

The vertical motion can then be explained by a displacemnet $y = d$, and the motion is governed by free fall:
$$
y = \frac{1}{2}g \cdot t^2
$$
when we substitute $y = 3d$ we get:
$$
3d = \frac{1}{2}g \cdot t^2
$$
we can now solve for t, which gives us
$$
t = \sqrt{\frac{6d}{g}}
$$
The horizontal displacement $x=3d$ and horizontal velocity is $v$, so:
$$
x = v \cdot t
$$
substituting for $x=3d$ and t
$$
3d = v \cdot \sqrt{\frac{6d}{g}}
$$
We can now solve for v
$$
v = \frac{3d}{\sqrt{\frac{6d}{g}}} = 3d\sqrt{\frac{g}{6d}}
$$
Write the square root as
$$
\sqrt{\frac{g}{6d}} = \frac{\sqrt{g}}{\sqrt{6d}}
$$
we now have
$$
v = \frac{3d\sqrt{g}}{\sqrt{6d}}
$$
Notice that $3d=\sqrt{9d^2}$, so
$$
v = \frac{\sqrt{9d^2g}}{\sqrt{6d}} = \sqrt{\frac{9dg}{6}} = \sqrt{\frac{3dg}{2}} = \frac{\sqrt{3dg}}{\sqrt{2}} 
$$

## Problem 3
![image.png](../../utils/kinematics-assignments-images/problem-3-2.png)

### Answer
From circular motion, centripetal acceleration is given by:
$$
a_{rad} = \frac{v^2}{R}
$$
The velocity of a point at the equator due to Earth's rotation:
$$
v = \frac{2 \pi R}{T}
$$

In [23]:
import math

# Given values
R = 6.37e6  # Radius of Earth in meters
T = 86400  # Rotation period in seconds

# Compute velocity at the equator
v = (2 * math.pi * R) / T

# Compute centripetal acceleration
a_rad = v**2 / R

a_rad # m/s^2

0.03368772507016205

A person at the equator becomes weightless when the centripetal acceleration equals the gravitational acceleration:
$$
a_{rad} = g
$$
From the equation:
$$
\frac{4 \pi^2 R}{T^2} = g
$$
Solving for T:
$$
T = \sqrt{\frac{4 \pi^2 R}{g}}
$$

In [24]:
# Given gravitational acceleration
g = 9.81  # m/s²

# Compute the required period for weightlessness
T_weightless = math.sqrt((4 * math.pi**2 * R) / g)

T_weightless / 60 / 60  # Convert to hours


1.406412054308739

## Problem 4
![image.png](../../utils/kinematics-assignments-images/problem-4-2.png)

### Answer

In [25]:
t = 3.5 # s
theta = 50 # degree
theta = math.radians(theta)  # Convert to radians
g = 9.81 # m/s^2

We know that the displacement vector from $A$ to $B$, over the $3.5$s, has a direction of $50\degree$ below the horizontal, meaning
$$
\tan{50\degree}=\frac{y}{x}
$$
where:
* $x = v_{0x}\cdot t = v_0 \cdot t$ (since the launch is horizontal)
* $y = \frac{1}{2}gt^2$ (vertical displacement due to gravity, downward)

we can now say
$$
\tan{50\degree} = \frac{\frac{1}{2}gt^2}{v_0 \cdot t} \iff v_0 = \frac{\frac{1}{2}gt}{\tan(50\degree)}
$$

In [26]:
# we compute the starting velocity
v0 = (1/2 * g * t) / tan(theta)
v0


14.4052429182360

## Problem 5
![image.png](../../utils/kinematics-assignments-images/problem-5-2.png)

In [27]:
H = 0.3 # m
g = 9.81 # m/s^2
L = 10 # m
R = L

### Answer
For this we will use the max height formular
$$
H = \frac{v_0^2\sin{\theta}^2}{2g} \iff v_0^2\sin{\theta}^2 = 2Hg
$$

We can now use the range formular
$$
R = \frac{v_0^2\sin{2\theta}}{g} \iff v_0^2\sin{2\theta} = Rg
$$
we can now solve this as we have only two unknown variables

In [28]:
v0, theta = symbols('v0 theta')

eq1 = Eq(v0**2 * sin(theta)**2, 2 * g * H)
eq2 = Eq(v0**2 * sin(2*theta), R*g)
# Solve the equations for v0 and theta
solution = solve((eq1, eq2), (v0, theta))
# We only want the positive solution for v0 and theta
solution = [(v0_val.evalf(), math.degrees(theta_val.evalf())) for v0_val, theta_val in solution if v0_val > 0 and theta_val > 0]
# Display the solution
solution

[(20.3626127989509, 6.84277341263094)]

For the time of flight we can just use horizontal motion:
$$
x = v_0\cos{\theta}\cdot t \iff t = \frac{x}{v_0\cos{\theta}}
$$
where $x = 10m$

In [29]:
x = 10 # m
v0 = solution[0][0]  # Extract the first solution for v0
theta = math.radians(solution[0][1])  # Convert theta to radians
# Compute the time of flight for the projectile in seconds
t = x / (v0 * cos(theta))
t


0.494619366829498

## Problem 6
![image.png](../../utils/kinematics-assignments-images/problem-6-2.png)

### Answer
For the assignment we are given:
* initial vertical position $y_0 = 3m$
* maximum height above the springboard $\Delta y = 2.5m \to$ total maximum height $y_{max} = y_0 + \Delta y = 3m + 2.5m = 5.5m$
* Final vertical position $y = 0m$ (the water surface)
* Horizontal range $R = 2.8m$

To find the initial velocity $v_0$ and launch angle $\theta$, we can use the following equations:
$$
y_{max} = y_0 + \frac{(v_0 \sin{\theta)^2}}{2g}
$$
$$
R = \frac{v_0^2 \sin{2\theta}}{g}
$$

For the horizontal velocity component we can use:
$$
x(t) = v_0 \cos{\theta} \cdot t
$$
For the total initial speed and angle we now have
$$
v_0 = \sqrt{v_{0x}^2 + v_{0y}^2}
$$
and
$$
\theta = \tan^{-1}\left(\frac{v_{0y}}{v_{0x}}\right)
$$


In [34]:
# Known quantities
y0 = 3          # m
delta_y = 2.5   # m
y_max = y0 + delta_y  # 5.5 m
g = 9.81        # m/s^2
R = 2.8         # m (horizontal distance to landing)
y = 0           # final y position

# Unknowns
v0, theta, t = symbols('v0 theta t', real=True, positive=True)

# Step 1: Use max height to get v0*sin(theta)
v0y = v0 * sin(theta)
eq1 = Eq(y_max, y0 + (v0y**2) / (2 * g))  # vertical motion max height

# Solve eq1 for v0*sin(theta)
v0y_val = [i for i in solve(eq1, v0y) if i > 0][0]  # Only positive root makes physical sense

# Step 2: Time of flight using vertical motion equation
# y = y0 + v0*sin(theta)*t - 0.5*g*t^2 = 0
eq2 = Eq(y0 + v0y_val * t - 0.5 * g * t**2, 0)

# Solve eq2 for time of flight
t_val = solve(eq2, t)
t_val = [sol.evalf() for sol in t_val if sol.evalf() > 0][0]  # positive root

# Step 3: Use horizontal motion to get v0*cos(theta)
# x = v0*cos(theta)*t = R → solve for v0*cos(theta)
v0x = R / t_val

# v0 = sqrt(v0x^2 + v0y^2)
v0_val = sqrt(v0x**2 + v0y_val**2).evalf()

# theta = atan(v0y / v0x)
theta_val = math.degrees(atan(v0y_val / v0x))

# Final results
print(f"v0y = {v0y_val:.1f} m/s")
print(f"Time of flight = {t_val:.1f} s")
print(f"v0x = {v0x:.1f} m/s")
print(f"Initial speed v0 = {v0_val:.1f} m/s")
print(f"Angle of projection θ = {theta_val:.1f} degrees")

v0y = 7.0 m/s
Time of flight = 1.8 s
v0x = 1.6 m/s
Initial speed v0 = 7.2 m/s
Angle of projection θ = 77.3 degrees


## Problem 7
![image.png](../../utils/kinematics-assignments-images/problem-7-2.png)

### Answer
We can start by setting the relative motion to the ground frame.
* horizontal velocity: v (inherited from the cart)
* vertical velocity: v (from the cannon shooting the projectile)

So the total initial velocity of the projectile is:
* $v_{0x} = v$
* $v_{0y} = v$

Since the projectile is launched vertically (in the cart frame) and only affected by gravity vertically:
* Time to reach the maximum height: $t_{up} = \frac{v_{0y}}{g} = \frac{v}{g}$
* Total time of flight: $t_{total} = 2 \cdot t_{up} = \frac{2v}{g}$
While in the air, the projectile travels horizontally at constant speed v:
$$
Horizontal \ distance = v_{0x} \cdot t_{total} = v \cdot \frac{2v}{g}
$$
We are told that this must equal the horizontal distance L:
$$
L = v \cdot \frac{2v}{g} \iff v^2 = \frac{gL}{2} \iff v = \sqrt{\frac{gL}{2}}
$$

## Problem 8
![image.png](../../utils/kinematics-assignments-images/problem-8-2.png)

### Answer
Since the throw and catch occur at the same height, potential energy is the same — the only thing affecting arrival speed is the trajectory.
* The minimum speed at the target occurs when the ball is thrown at a $45\degree$ angle.

Why $45\degree$?
* For a given range $R = \frac{v_0^2 \sin{2\theta}}{g}$, the maximum range occurs at $\theta = 45\degree$ because $\sin{90\degree} = 1$ and this is the maximum value of the sine function.

So what we can do is use the range formula to find the minimum speed:
$$
R = \frac{v_0^2 \sin{90\degree}}{g} \iff v_0^2 = Rg \iff v_0 = \sqrt{Rg}
$$

## Problem 9
![image.png](../../utils/kinematics-assignments-images/problem-9-2.png)

### Answer
We are given:
* Particle 1 is projected horizontally from a height h with speed $v_0$
* Particle 2 moves horizontally at speed $\frac{v_0}{2}$ from a point distance L away
* They start at the same time and collide at time T

We can start by analyzing the horizontal motion.
* First we place the origin at the point where particle 1 is launched.
* Particle 1's horizontal position at time T is:
$$
x_1 = v_0 \cdot T
$$
* Particle 2 starts from x = L and moves left:
$$
x_2 = L + \frac{v_0}{2} \cdot T
$$
They collide when $x_1 = x_2$:

In [46]:
x1, x2, v0, T, L = symbols('x1 x2 v0 T L')
eq = Eq(v0 * T, L + (v0 / 2) * T)
# Solve for T
T_solution = solve(eq, T)[0]  # Get the first solution
T_solution

2*L/v0

We can now determine the height h. From vertical motion of particle 1 we have that it falls under gravity for time T:
$$
h = \frac{1}{2}gT^2
$$

In [47]:
g = symbols('g')
h = 1/2 * g * T_solution**2
h

2.0*L**2*g/v0**2

The value of h is now known. The velocity in the x- and y-directions of the first particle is
$$
v_{1x} = v_0, \quad v_{2y} = -gt
$$
The collision occurs at time $T$
$$
h = \frac{1}{2}gT^2 \iff T = \sqrt{\frac{2h}{g}}
$$
The velocity of the particle 1 before the collision is:
$$
v_{1x} = v_0, \quad v_{1y} = -gT = -\sqrt{2gh}
$$
The velocity vector $\vec{v}_1$ is:
$$
\vec{v}_1 = \begin{pmatrix} v_0 \\ -\sqrt{2gh} \end{pmatrix}
$$
The second particle has only a velocity in the x-direction:
$$
\vec{v}_2 = \begin{pmatrix} \frac{v_0}{2} \\ 0 \end{pmatrix}
$$
The relative velocity between the two particles can now be calculated
$$
\vec{v}_{rel} = \vec{v}_1 - \vec{v}_2 = \begin{pmatrix} v_0 - \frac{v_0}{2} \\ -\sqrt{2gh} - 0 \end{pmatrix} = \begin{pmatrix} \frac{v_0}{2} \\ -\sqrt{2gh} \end{pmatrix}
$$
The magnitude of the relative velocity vector is:
$$
|\vec{v}_{rel}| = \sqrt{\left(\frac{v_0}{2}\right)^2 + \left(-\sqrt{2gh}\right)^2} = \sqrt{\frac{v_0^2}{4} + 2gh}
$$


## Problem 10
![image.png](../../utils/kinematics-assignments-images/problem-10-2.png)

### Answer
**a)**

![image.png](../../utils/kinematics-assignments-images/sketch-problem-10.png)

**b)**

The parabola $y(x)$ is given by the equation:
$$
y(x) = tan(\theta) \cdot x - \frac{g}{2v_0^2 \cos^2(\theta)} x^2
$$
We can now use python to isolate the initial velocity $v_0$:

In [56]:
y, x, g, theta, v0 = symbols('y x g theta v0')
eq = Eq(y - tan(theta) * x, - (g / (2 * v0**2 * cos(theta)**2)) * x**2)
sol = solve(eq, v0)[1]
# A path of motion that travels above the defenders and into the goal is found by insert
v0 = sol.subs([(x, 10), (y, 2), (g, 9.81), (theta, math.radians(15))])  # Example values
v0.evalf()  # Evaluate the expression to get a numerical value

27.8152890877933

**c)**

We calculate the initial velocity for the two extreme cases where the ball lands on the goal line $(x,y) = (3L, 0)$, and where it hits the goal post $(x,y) = (3L, H)$.
When the ball hits goal line $(x,y) = (3L, 0)$, we have:

In [58]:
v0_new = sol.subs([(x, 3*10), (y, 0), (g, 9.81), (theta, math.radians(15))])  # Example values
v0_new.evalf()  # Evaluate the expression to get a numerical value


24.2610799429869

We can now notice that the ball will collide with the wall of defenders, see question b).

In [59]:
v0_goal_post = sol.subs([(x, 3*10), (y, 2.44), (g, 9.81), (theta, math.radians(15))])  # Example values
v0_goal_post.evalf()  # Evaluate the expression to get a numerical value

29.0711412249509

This means that the velocity of the ball should be in the interval
$$
27.8 \frac{m}{s} < v_0 < 29.1 \frac{m}{s}
$$
This is a small interval, so we can conclude that it is not an easy goal to make

## Problem 11
![image.png](../../utils/kinematics-assignments-images/problem-11-2.png)

### Answer
We know the tangential acceleration is given by:
$$
a_t = \frac{dv}{dt} = a_0
$$
We integrate with respect to time
$$
\int_0^{v_0} dv = \int_0^t a_0 dt \implies v_0 = a_0 t
$$
The radial acceleration is given by the centripetal acceleration.
$$
a_r = \frac{v^2}{R} = \frac{(a_0 t)^2}{R} = \frac{a_0^2 t^2}{R}
$$

**b)**

The tangential acceleration is constant during the circular motion, but the radial acceleration is
increasing with $t$. We must determine the time $t$ it takes to perform one revolution.

The motion is along the line which is a 1-dimensional movement with length $2\pi R$ and constant acceleration $a_0$.
$$
x(t) = \frac{1}{2} a_0 t^2
$$
After one revolution $x(T)$, the distance is $2\pi R$:
$$
2\pi R = \frac{1}{2} a_0 T^2 \implies T = 2\sqrt{\frac{\pi R}{a_0}}
$$
The radial acceleration after one revolution (after time T has passed) can be determined
$$
a_r(T) = \frac{a_0^2 T^2}{R} = \frac{a_0^2 (2\sqrt{\frac{\pi R}{a_0}})^2}{R} = \frac{4\pi a_0 R}{R} = 4\pi a_0
$$
We now have the two components of the acceleration to find the magnitude
$$
a = \sqrt{a_t^2 + a_r^2} = \sqrt{a_0^2 + (4\pi a_0)^2} = a_0 \sqrt{1 + 16\pi^2}
$$